server/mqtt_test.go

```diff
@@ -7546,6 +7545,91 @@ func TestMQTTDecodeRetainedMessage(t *testing.T) {
 	testMQTTExpectNothing(t, r)
 }
 
+func TestMQTTSparkbBirthHandling(t *testing.T) {
+	o := testMQTTDefaultOptions()
+	s := testMQTTRunServer(t, o)
+	defer testMQTTShutdownServer(s)
+
+	// Publish an NBIRTH message. Make sure it is received as both the original
+	// subject, and as a $sparkplug/certificates one. Since BIRTH messages are
+	// QoS0, use a NATS client/subscriber to monitor.
```

Comment:
I don't see that you are using NATS subscriptions to monitor in this test.

Reply:
Oops, sry was leftover from the first implementation.